### DISCUSSION TITLE
Boxplot: How does it work?

### DISCUSSION THREAD

**Discussion by XiangRongLin** [getredash/redash#6805] [Q&A]
I don't understand how boxplot works. I can only find these 2 sites mentioning it, but not explaining what format the data needs to be
1. https://redash.io/help/user-guide/visualizations/visualization-types
2. https://redash.io/help/user-guide/visualizations/chart-visualizations

for the line charts i usually have a date and value, but for the "box chart" visualiziation I dont see the "candles" as shown in the sites.

**Comment by ltrainstg** [comment]
Typically boxplots show the 1st and 3rd interquartile range and plot the median as the center. See https://en.wikipedia.org/wiki/Box_plot. 
I don't know what you mean by "candles". 

Typically a box plot would be done to summarize data and show probably outlies at a given point in time. 
So I would expect data like rows are people and a column is height, weight, age ect. Then you'd get a box plot on height, weight, ect and see the distribution. 

It's not really used to plot change over time, but occasionally you'll see Q1 Q2 Q3 Q4 boxplots to see if a distribution changes over times.

**Comment by XiangRongLin** [comment]
I meant it as "how does it work in redash", since I could find no way to create a boxplot.

But it already resolved itself. Boxplot is deprecated and one can't create deprecated visualizations through the UI anymore. One can hack it themselve, by sending the corresponsing HTTP request, but the visualization is not performant enough for my usecase with way too much data.
